packages/gatsby/src/utils/babel-config.js

```diff
@@ -134,55 +57,45 @@ module.exports = async function babelConfig(program, stage) {
 
   // If user doesn't have a custom babelrc, add defaults.
   if (!babelrc) {
-    babelrc = {}
-  }
-  if (!babelrc.plugins) {
-    babelrc.plugins = []
-  }
-  if (!babelrc.presets) {
-    babelrc.presets = []
+    babelrc = {
+      cacheDirectory: true,
+      plugins: [],
```

Comment:
should probably include class properties

Reply:
Looking at CRA's babel 7 upgrade PR – just copy their plugin list I think? https://github.com/facebook/create-react-app/pull/3522/files#diff-4dea0a44b1f2f65c2294fd4dae2e25f7R72